src/Access/Common/RowPolicyDefs.h

```diff
@@ -43,6 +43,12 @@ enum class RowPolicyFilterType
     MAX
 };
 
+enum class RowPolicyOrderType
```

Comment:
Why is it called "order type"? It doesn't seem it has something to do with the "order"

Reply:
This is about main meaning of order, i.e. restrictive order vs permissive order. However, in latest version I got rid of types.